docs/source/user_guides/context_guide.rst

```diff
@@ -154,17 +154,6 @@ Public methods
 * **pipeline**: Return ``Pipeline`` object that is used to process this context.
   This can be used to get ``Script``, ``start_label`` or ``fallback_label``.
 
-Private methods
-^^^^^^^^^^^^^^^
-
-These methods should not be used outside of the internal workings.
-
-* **set_last_response**
-* **set_last_request**
-* **add_request**
-* **add_response**
-* **add_label**
-
```

Comment:
The difference between the two is that `ctx.labels[ctx.current_turn_id]` is a coroutine and I didn't want `last_...` kind of properties to become async.

That reminds me that for this workaround to work we need to make subscript value at least `1` (so that it always loads the last turn).